pom.xml

```diff
@@ -63,25 +63,19 @@ THE SOFTWARE.
     </pluginRepositories>
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.444</jenkins.version>
+        <jenkins.version>2.452.2</jenkins.version>
```

Comment:
switch to next available LTS line

Reply:
I thought that we usually set the ".1" as the base release when choosing the current LTS baseline.  That's what is suggested in ["Choosing a Jenkins version"](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#currently-recommended-versions).

Reply:
```suggestion
        <jenkins.version>2.452.1</jenkins.version>
```

Reply:
dbca4ef76ab35a9903aa0a57f3820b3ad1fc0772

Reply:
reintroduced with b16da6576c35e787186cdd7ceb0e9775a6a0e8d9